design-proposals/network-binding-plugin/network-binding-plugin.md

```diff
@@ -1258,3 +1274,50 @@ Pros:
 Cons:
 - Requires an API change.
 - When upgrading KubeVirt / network binding plugin versions, the sidecar container's resource specification might require adjustments.
+
+# Appendix H:  Alternatives to compute container resource overhead specification
+
+1. Manually setting the VM's resources:
+
+The user can override KubeVirt's algorithms and set resource requirements.
+
+Pros:
+- Already implemented.
+
+Cons:
+- Error prune
+- The user does not take into account the overhead considerations KubeVirt takes when templating a virt-launcher pod.
+
+2. Additional API for compute container resource overhead:
+
+The network binding plugin API in the KubeVirt CR could receive an additional input field to specify the resource requirements overhead for the compute container:
+
+```yaml
+apiVersion: kubevirt.io/v1
+kind: KubeVirt
+metadata:
+  name: kubevirt
+  namespace: kubevirt
+spec:
+  configuration:
+    network:
+      binding:
+        mynetbindingplugin:
+          sidecarImage: quay.io/kubevirt/mynetbindingplugin
+          computeContainerResourceOverhead:
```

Comment:
Same here, I would try to avoid hard coding the container name in the field name.

Reply:
Same answer as before.